app/services/idv/profile_maker.rb

```diff
@@ -1,18 +1,31 @@
 module Idv
   class ProfileMaker
-    attr_reader :pii_attributes, :profile
+    attr_reader :pii_attributes
 
     def initialize(applicant:, user:, normalized_applicant:, vendor:, phone_confirmed:)
-      @profile = Profile.new(user: user, deactivation_reason: :verification_pending)
-      @pii_attributes = pii_from_applicant(applicant, normalized_applicant)
+      self.pii_attributes = pii_from_applicant(applicant, normalized_applicant)
+      self.user = user
+      self.vendor = vendor
+      self.phone_confirmed = phone_confirmed
+    end
+
+    def make_profile
```

Comment:
what if we renamed this to `save_profile` -- since  to me,`make` sounds more like a `#new` than a `#save!`?